src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -403,7 +403,7 @@ export default {
                   }
                 }
                 // use default Priority and Waive Fees flags
-                this.updateFilingData('add', FilingCodes.ADDRESS_CHANGE_OT, this.isPriority, this.isWaiveFees)
+                this.updateFilingData('add', FilingCodes.ADDRESS_CHANGE_BC, this.isPriority, this.isWaiveFees)
```

Comment:
Add a comment explaining that this only applies to BC (re: your question in RocketChat).

Reply:
Updated in next commit

Reply:
Thanks for the comment but I find it hard to understand.

Reply:
Fair enough! This line of code is within a conditional that is inside of a method for structuring the filing from a draft. So within the conditional it identifies records office, which is an exclusive property to BCOMPs filings (at the moment) and updates the filingData accordingly. 
Does the context help or should i reword the comment?

Reply:
LOL, that's worse :>

I'm looking for a comment that helps me understand that this code is for "BCOMP only" (for example) and that the programmer wrote it on purpose (instead of, say, forgetting to update this code).

Reply:
I see, ok :) I'll take another shot at it 😆

Reply:
"code should be easy to review"
(ie, understand)

Reply:
OK, looks good.

Reply:
For sure, though i'm a little confused on how this update in particular isn't clear. Was it the code wrapping this method that is confusing?